community/bareos/APKBUILD

```diff
@@ -2,93 +2,94 @@
 # Contributor: Francesco Colista <fcolista@alpinelinux.org>
 # Maintainer: Francesco Colista <fcolista@alpinelinux.org>
 pkgname=bareos
-pkgver=17.2.7
-pkgrel=1
+pkgver=18.2.6
+pkgrel=2
```

Comment:
I think it should be 0

Reply:
Whoops, forgot to change this. Fixed, thanks.